### DISCUSSION TITLE
Where to find Ed25519 public keys for self hosting solution?

### DISCUSSION THREAD

**Discussion by sharadkak** [keygen-sh/community#164] [General]
Hi, currently I am self hosting Keygen to manage licenses, while for Keygen dashboard one can download Ed25519 public keys from settings to verify signature to implement node locked licenses, where can I find the same if I am self hosting keygen? Is there any API request I have to make to find it?
Thanks!

**Comment by ezekg** [comment]
From the console, run:

```rb
Account.first.ed25519_public_key
```
